### PR TITLE
Feature/product bumper

### PR DESCRIPTION
**Pull request by oklinov**

### CODE REVIEW

File: fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java [hunk 1]
```diff
@@ -170,6 +179,43 @@
...
+                    getJDKVersion(toProduct).flatMap(toJDK ->
+                            extractProjectIds(paramsMap).flatMap(projectIds ->
+                                    checkProjectIds(projectIds).flatMap(projects ->
+                                            modifyJobs(
+                                                    projects,
+                                                    new ProductBumper(
+                                                            fromProduct.getPackageName(),
+                                                            toProduct.getPackageName(),
+                                                            fromJDK,
+                                                            toJDK
```
Comment (judovana) [inline fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java:196]: kokos. stream of strems using lamdas^lamdatou

Comment (judovana) [reply fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java:196]: Seriously, this sh ould be rewrtten

File: fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java [hunk 2]
```diff
@@ -217,8 +217,10 @@
     }
 
     public String getHelp() {
-        return ""
-                + MISC + '/' + BUMP + PLATFORMS + "?from=[platformId]&to=[platformId]&projects=[projectsId1,projectId2,..projectIdN]\n";
+        final String prefix = MISC + '/' + BUMP;
+        return "\n"
```
Comment (judovana) [inline fake-koji/src/main/java/org/fakekoji/api/http/rest/BumperAPI.java:221]: I had pushed this minifix few minites efore your PR:*

**Comment by judovana** [comment]
Ok to go after rebase.

**Comment by oklinov** [comment]
Looks good

**Pull request merged**